tests/integration/conftest.py

```diff
@@ -500,6 +576,86 @@ async def go_non_root_app_fixture(
     return app
 
 
+@pytest.mark.skip_juju_version("3.4")
+@pytest.fixture(scope="module", name="expressjs_app")
+def expressjs_app_fixture(
+    juju: jubilant.Juju,
+    pytestconfig: pytest.Config,
+):
+    """ExpressJS charm used for integration testing.
+    Builds the charm and deploys it and the relations it depends on.
+    """
+    app_name = "expressjs-k8s"
+
+    use_existing = pytestconfig.getoption("--use-existing", default=False)
+    if use_existing:
+        yield App(app_name)
```

Comment:
If there is no cleanup that is required, i think we can just return the App right away :)

```suggestion
        return App(app_name)
```